src/test/unit/algorithms/Cells4Test.cpp

```diff
@@ -179,94 +175,7 @@ TEST(Cells4Test, pickleSerialization) {
   }
 }
 
-TEST(Cells4Test, capnpSerialization) {
-  Cells4 cells(10, 2, 1, 1, 1, 1, 0.5, 0.8, 1, 0.1, 0.1, 0, false, -1, true,
```

Comment:
Note the test just before this one is pickleSerialization. This is the same test but using binary serialization.  I renamed it just Serialization because it is pickling only in the Python.